### DISCUSSION TITLE
Self-hosted runner under docker and Ubuntu20.04 with checkout actions error: OCI runtime exec failed: exec failed: unable to start container process: exec: "/__e/node16/bin/node": stat /__e/node16/bin/node: no such file or directory: unknown

### DISCUSSION THREAD

**Discussion by linxter** [community/community#83008] [Actions]
### Select Topic Area

Bug

### Body

Dear community,

I am trying to setup self-hosted runner for GitHub Actions under docker in docker with Ubuntu20.04LTS and I have issue with checkout actions (uses: actions/checkout@v3): 
`OCI runtime exec failed: exec failed: unable to start container process: exec: "/__e/node16/bin/node": stat /__e/node16/bin/node: no such file or directory: unknown`

If I run self-hosted runner without docker then checkout (uses: actions/checkout@v3) works well, but if I add self-hosted runners under docker I get error above.

For setup self-hosted runners under docker I used mount follow volume in compose config file:
```
volumes:
      - /var/run/docker.sock:/var/run/docker.sock

```
Main idea - create docker image under docker-compose which will be run a lot of self-hosted actions-runners on one server which add possibility to run **workflow jobs in parallel on one server**.

What I need to add to config for fix issue with actions/checkout@v3 and node16?

Regards,
Arthur German.

**Comment by muhammadhamzasajjad** [comment]
Having the same issue.

**Comment by neethalokanath** [comment]
is this issue solved ?
